modules/text-input/react/lib/TextInput.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
-import styled from 'react-emotion';
-import {GrowthBehavior, ErrorType, errorRing} from '@workday/canvas-kit-react-common';
-import {inputColors, spacingNumbers, type} from '@workday/canvas-kit-react-core';
+import {GrowthBehavior, ErrorType} from '@workday/canvas-kit-react-common';
+import {inputColors} from '@workday/canvas-kit-react-core';
 import {SystemIcon} from '@workday/canvas-kit-react-icon';
 import {exclamationCircleIcon, exclamationTriangleIcon} from '@workday/canvas-system-icons-web';
 import InputIconContainer from './InputIconContainer';
+import Input from '../../../input/react/index';
```

Comment:
This import should match the others in this file:

```suggestion
import Input from '@workday/canvas-kit-react-input';
```